### DISCUSSION TITLE
@nx/enforce-module-boundaries allow to disable the nx "import of apps is forbidden" error

### DISCUSSION THREAD

**Discussion by OmerDavidson** [nrwl/nx#23494] [Feature Requests]
I am currently adding the @nx/enforce-module-boundaries eslint rule to my repo.
I have a project of type library that is importing a file from a project of type application.
Therefore I am getting the "import of apps is forbidden" error.
Unfortunately I have no way of going around this so I have to disable the eslint rule in that file.
I would like to be able to just ignore this rule in a specific project

**Comment by MilanKovacic** [comment]
This is already possible. You can simply override the rule in the project's eslint file.
https://discord.com/channels/1143497901675401286/1238389506911113216

**Reply by OmerDavidson** [reply]
I can't seem to open this link. Can you please provide an example for how to turn off that rule?

Thanks!

**Reply by OmerDavidson** [reply]
Just to clarify, I want to still have all the other errors. Just not the "import of apps is forbidden" error
